app/layouts/DashboardLayout/BalanceIndicator.view/BalanceIndicator.d.ts

```diff
@@ -1,6 +1,10 @@
 export interface BalanceIndicatorProps {
   balance: string;
-  importLedger: () => void;
+  containsUnverified: boolean;
+  getViewOnlySync: () => Promise<void>;
+  importLedger: () => Promise<void>;
+  importViewOnlySync: () => Promise<void>;
```

Comment:
Is there a reason they need to be defined as `() => Promise<void>` rather than `async () => undefined`? I notice they're called with the async/await pattern

Reply:
`async () => {}` is the syntax for declaring an async function. `() => Promise<thing>` is the syntax for typing an async function.